usb_autorunner_profiles/sysinfo/package_resources.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
Shouldn't this be `+e`? Since `+e` is the default, I suggest to actually simply remove this line.